File: examples/node/delete/deleteMany.js
const MongoClient = require('mongodb').MongoClient;
const assert = require('assert');

// Connection URL
const url = 'mongodb://localhost:27017/test';

// Database Name
const dbName = 'test';

// use connect method to connect to the server
MongoClient.connect(url, function(err, client){
  assert.equal(null, err);
  console.log("Connected successfully to server!");
  const db = client.db(dbName)

  // Delete many documents (example 56)
  var promise = 
  db.collection('inventory').deleteMany({ 
    status: "A" 
  })
  .then(function(result) {
    // process result
    console.log("deleteMany example result: " + result)
  })            
  // End deleteMany example
  client.close();
});

File: source/includes/examples/python/delete/deleteMany.py
# Start Connection
from pymongo import MongoClient
from pprint import pprint
import json

connection_string = "mongodb://testuser:<PASSWORD>@localhost:27017/test?authSource=admin"

client = MongoClient(connection_string)
# End Connection

# Start select db
db = client.test
# End select db

# Start deleteMany (Example 57)
result = db.inventory.delete_many({"status": "A"})
# End deleteMany (Example 57)

pprint(result.raw_result)

client.close()